src/components/ReanimatedSwipeable.tsx

```diff
@@ -144,31 +144,26 @@ export interface SwipeableProps
   onSwipeableCloseStartDrag?: (direction: 'left' | 'right') => void;
 
   /**
-   *
-   * This map describes the values to use as inputRange for extra interpolation:
-   * AnimatedValue: [startValue, endValue]
-   *
-   * progressAnimatedValue: [0, 1] dragAnimatedValue: [0, +]
+   * `progress`: Equals `0` when action is closed, `1` when action is open, linearly increases to `Infinity` as action overshoots it's open position.\
+   * `translation`: translation of swipeable in pixels.
```

Comment:
```suggestion
   * `translation`: translation of the swipeable in pixels.
```

Reply:
fixed in [6f68b59](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/6f68b5904f65ca9d81bbca4a8eb8e8d19f070f24)